src/averageAges.js

```diff
@@ -14,7 +14,20 @@
  *
  * @return {number}
  */
+
+const getAverageAge = list => list.reduce((sum, person) =>
+  sum + (person.died - person.born), 0)
+   / list.length;
+
 function calculateMenAverageAge(people, century) {
+  const men = people.filter(person => century
+    ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+    : person.sex === 'm');
+
+  const averageAge = getAverageAge(men);
+
+  return averageAge;
```

Comment:
if the variable is not used, the value can be returned immediately
```suggestion
  return getAverageAge(men);
```